rktlet/runtime/helpers.go

```diff
@@ -345,6 +355,17 @@ func generateAppAddCommand(req *runtimeApi.CreateContainerRequest, imageID strin
 	return cmd, nil
 }
 
+// maybeCreateHostPathVolume creates the source dir for Mount if it doesn't
+// exist since rkt doesn't do it. It returns whether the dir was created.
+func maybeCreateHostPathVolume(mount *runtimeApi.Mount) (created bool, err error) {
+	if _, err = os.Stat(mount.HostPath); os.IsNotExist(err) {
```

Comment:
What's the point of stat? I'd just call `MkdirAll` directly - it won't do anything if the directory is already there and will return some error in other cases.

Reply:
Good point. It might be a file but I guess we could ignore `ENOTDIR`. It seems the first think MkdirAll does is `Stat()` https://golang.org/src/os/path.go?s=488:535#L9

Reply:
The other thing is that for logging purposes we want to know if we created a directory or it was already there and `MkdirAll` doesn't give that information.

Reply:
Sure, sounds fine then.